src/utils.js

```diff
@@ -40,3 +40,11 @@ export function limitLen(s, cap) {
     return s;
   }
 }
+
+export function isIframe() {
```

Comment:
I can see usefulness of exposing this and testing e2e across browsers, that said, I don't know what's the propensity of this to break, so you decide Marten, because you did the research.

Reply:
Supposedly this is the safest and most widely supported way according to stackoverflow :P. 
Integration tests should cover this. Even if this breaks nothing would break for our users.